sample-app/notification-sample-code/nodejs/README.md

```diff
@@ -4,7 +4,9 @@ This sample code makes use of the Create Subscription endpoint to create a subsc
 
 ## Pre-requisites
 
-[Node.js](https://nodejs.org/en/) - Install node.js
+* Have an cloudlink account and obtain a client id
```

Comment:
Should we add that you can obtain more information on this through the [Developer Portal](https://developer.mitel.io/)?